### PR TITLE
feat: reset

### PR DESCRIPTION
**Pull request by cachho**
Resets the database and deletes all embeddings. Irreversible. Requires reinitialization afterwards.

This is a nice quality of live feature, compared to deleting the folder. Especially in docker this is a must.

### CODE REVIEW

**Comment by cachho** [comment]
PR for issue #36

**Comment by taranjeet** [comment]
@cachho : can you resolve the conflicts?

File: README.md [hunk 1]
```diff
@@ -314,8 +323,9 @@ This section describes all possible config options.
 *coming soon*
 
 #### **Query Config**
-
-*coming soon*
+|option|description|type|default|
+|---|---|---|---|
+|number_documents|provide multiple documents as context|int|1|
```
Comment (taranjeet) [inline README.md:328]: can we call it "number of documents to be retrieved as context"

File: embedchain/embedchain.py [hunk 2]
```diff
@@ -294,9 +302,16 @@ def dry_run(self, input_query):
         """
         if config is None:
             config = QueryConfig()
-        context = self.retrieve_from_database(input_query)
-        prompt = self.generate_prompt(input_query, context)
+        contexts = self.retrieve_from_database(input_query, config)
+        prompt = self.generate_prompt(input_query, contexts)
         return prompt
+
```
Comment (taranjeet) [inline embedchain/embedchain.py:308]: I think here, two functionalities are getting merged.

@cachho : can you keep this PR for number of docs in context, and create a new PR for reset or vice versa.

But the idea is to have different PR for both.

**Pull request closed without merging**